src/lib/containers/table/SynapseTable.tsx

```diff
@@ -369,22 +366,7 @@ export default class SynapseTable extends React.Component<
     const hasResults = (data.queryResult?.queryResults.rows.length ?? 0) > 0
     // Show the No Results UI if the current page has no rows, and this is the first page of data (offset === 0).
     if (!hasResults && queryRequest.query.offset === 0) {
-      if (queryRequest.query.additionalFilters) {
```

Comment:
`additionalFilters` will be used for cases other than column search, so this no longer makes sense (and prompted this entire PR)